docs/guides/monitoring/telegraf.md

```diff
@@ -0,0 +1,150 @@
+# Monitoring your CodiMD instance with telegraf
+
+CodiMD provides an easy way to monitor very basic metrics. Those can easily be
+recorded by [telegraf](https://github.com/influxdata/telegraf) for further use
+with - for example - [influxdb](https://www.influxdata.com/time-series-platform/influxdb/)
+and [grafana](https://grafana.com/).
+
+To collect the metrics, use the [http input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/http).
+It will convert all numeric values from json and store them in your database.
+
+![Example screnshot of CodiMD via Telegraf in Grafana](../images/monitoring/telegraf-grafana.png)
+
+```toml
+# /etc/telegraf/telegraf.d/codimd.conf
+[[inputs.http]]
+  interval = "1m" # strictly speaking this line is not even neccessary
+  urls = [
+    "https://your-codi-server/status" # change this!
+  ]
+  data_format = "json"
+```
+
+You will now start seeing the values in Grafana, as a start, you can use this
+panel:
```

Comment:
Mhm, sadly importing this into my test setup didn't really work :/

Used this setup: https://github.com/nicolargo/docker-influxdb-grafana.git